lambda-resume-builder/src/Components/EndorsementChecklist/EndorsementChecklist.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+
+const EndorsementChecklist = () => {
+   
+        return (
+            <h1>This Is The Endorsements Checklist Component</h1>
+        )
+    
+        
+    
+    
+    
```

Comment:
### See, now this is a great example of properly using white space to emphasize the importance of a component. 
Something tells me that we might to consider refactoring this into a class component instead of functional though. It just doesn't seem like this will go anywhere otherwise.